lib/crowbar/client/command/proposal/create.rb

```diff
@@ -103,6 +103,11 @@ def from_file
             rescue
               err "Failed to process file"
             end
+            if json["id"] != args.proposal
+              say "id(#{json["id"]}) in file and name(#{args.proposal}) do not match"
```

Comment:
I'm not sure but maybe the message is a bit too much cryptic. What about: `Given proposal name and file proposal name do not match, using #{json["id"]}.`? Then you could also drop the second `say`.